openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -140,24 +143,23 @@ def __init__(self):
 
         # check if the toolkit can be loaded
         if not self.is_available():
-            msg = (
-                f"The required toolkit {self._toolkit_name} is not "
-                f"available. {self._toolkit_installation_instructions}"
-            )
             if self._is_installed is False:
-                raise ToolkitUnavailableException(msg)
+                raise ToolkitUnavailableException(
+                    "OpenEye Toolkits are not installed."
+                    "{self._toolkit_installation_instructions}"
```

Comment:
BLOCKING: Second string is not an f-string - it's probably easier and nicer to just concatenate these.

```suggestion
                    "OpenEye Toolkits are not installed."
                    + self._toolkit_installation_instructions
```